README.md

```diff
@@ -68,6 +113,32 @@ So the version must be a valid tag in <https://github.com/anidl/multi-downloader
 
 ```bash
 #!/bin/bash
-VERSION=2.0.18
-docker build $(pwd) --build-arg VERSION=$VERSION -t anidl:$VERSION
+VERSION=$(cut -d'-' -f 1 VERSION)
+docker build . --build-arg VERSION=$VERSION -t anidl:$VERSION
+```
+
+Note: The version in [VERSION](VERSION) includes a suffix to track the image version so we strip that off before passing it to the build.
+
+With BuildKit:
+
+```bash
+docker buildx build . -t anidl
+```
+
+```bash
+#!/bin/bash
+VERSION=$(cut -d'-' -f 1 VERSION)
+docker buildx build . --build-arg VERSION=$VERSION -t anidl:$VERSION
 ```
+
+## Official Image
+
+Since creating this, the source project has added it's own image.
+
+[Docker Hub](https://hub.docker.com/r/izuco/multi-downloader-nx/)
+
+[Dockerfile](https://github.com/anidl/multi-downloader-nx/blob/master/Dockerfile)
+
+As far as I can tell, both images serve the same end. I'll attempt to keep this updated anyways.
+
+<sup>Mines smaller :smirk:</sup>
```

Comment:
Mines official 😜

Reply:
Let me know if there's anything you think is worth contributing! I didn't initially because I have a kind of "don't step on their toes" approach to open source.

Reply:
When you have something you want to add or you see an issue that you can fix go for it.

Just comment that you are working on it :)